public_suffix_list.dat

```diff
@@ -12518,10 +12518,6 @@ feste-ip.net
 knx-server.net
 static-access.net
 
-// cPanel L.L.C. : https://www.cpanel.net/
-// Submitted by Dustin Scherer <public.suffix@cpanel.net>
-*.cprapid.com
```

Comment:
> dig TXT +short _psl.cprapid.com                                                                                                                                                                                                            
"https://github.com/publicsuffix/list/pull/1892"

Could you please update this, since you update the entry?

Reply:
a good point, to make the entry current in the DNS.   We typically push for this where updater is not same as submitter, to ensure administrative ownership/admin of domain.

in this case, the submitter is 'known', so just trusting the existence of a pull request number (even if related to prior request) is ok...